src/main/java/org/jasig/portlet/widget/links/ResourceLink.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.jasig.portlet.widget.links;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Data object for Resource Links.They have the following Attributes:
+ *  - title
+ *  - description
+ *  - icon
+ *  - url
+ *  - groups
+ */
+public class ResourceLink {
+
+    private static final Logger log = LoggerFactory.getLogger(ResourceLink.class);
```

Comment:
The sources I've encountered on this topic suggest that these objects are better as non-static...

```
[private | protected] final Logger logger = LoggerFactory.getLogger(getClass());
```
For my part, I usually prefer `private`.

Reply:
`static` required to use the logger in static methods. But since we generally avoid it, I will get out of the habit of using it :)